<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Research</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 64px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h2 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h3 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h4 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h5 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h6 {
  padding-top: 69px;
  margin-top: -69px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Brian D. Williamson</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="research.html">
    <span class="fa fa-flask"></span>
     
    Research
  </a>
</li>
<li>
  <a href="teaching.html">
    <span class="fa fa-chalkboard"></span>
     
    Teaching
  </a>
</li>
<li>
  <a href="software.html">
    <span class="fa fa-file-code"></span>
     
    Software
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="contact.html">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/bdwilliamson">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://www.linkedin.com/in/brianwilliamson26/">
    <span class="fa fa-linkedin fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Research</h1>

</div>


<p>I am interested in problems of high-dimensional data, data science, and statistical inference. In particular, I enjoy working on developing new methodology for addressing scientific questions using large and complex datasets, and appropriately quantifying uncertainty in the resulting estimates. I also enjoy working on problems involving how best to tell a story using data. I have been primarily motivated by applications in public health and medicine, but I am always open to collaborations on interesting problems in any area of research.</p>
<p>To date, I have been primarily focused on developing a model-agnostic measure of variable importance. This allows estimates of importance to be compared across different machine learning algorithms. The work develops a rigorous framework for estimating and performing valid statistical inference on the true importance. While the procedure is broadly applicable, one area in which we have employed the procedure is the study of vaccine efficacy. Our proposed approach allows investigators to use state-of-the-art machine learning tools to estimate important variables for predicting when a vaccine is effective. This, in turn, may help to more quickly develop new candidate vaccines for a range of diseases.</p>
<p>For a complete list of my scholarly works, please see my <a href="https://scholar.google.com/citations?user=k4RWx44AAAAJ&amp;hl=en">Google Scholar page</a>.</p>
<div style="text-align: center">
<figure>
<img src="images/ROC_curve_with_Env_inset_v2.png" alt="ROC curves and important amino acid sequence positions for a proposed VRC01 neutralization sensitivity predictor." width="400" height="400" />
<figcaption>
ROC curves for a proposed VRC01 neutralization sensitivity predictor. Inset: 3D structure of the HIV Envelope protein with colored areas based on identified important features and groups using our proposed variable importance measure. Please see below, under “Broadly neutralizing antibodies”, for more detail.
</figcaption>
</figure>
</div>
<div id="statistical-methods" class="section level1">
<h1>Statistical methods</h1>
<div id="variable-importance" class="section level2">
<h2>Variable importance</h2>
<p>It is often of interest to understand the population interplay between the outcome and measured covariates. Variable importance is a useful tool towards this goal. Examples include: HIV vaccine studies, correlates of risk analyses, and identifying patients at high risk of returning to a hospital after discharge for heart failure. This is joint work with <a href="http://faculty.washington.edu/mcarone/about.html">Marco Carone</a>, <a href="https://www.fredhutch.org/en/labs/profiles/gilbert-peter.html">Peter Gilbert</a>, and <a href="http://faculty.washington.edu/nrsimon/">Noah Simon</a>.</p>
<p><a href="http://biostats.bepress.com/uwbiostat/paper422/">preprint</a> | <a href="https://github.com/bdwilliamson/vimp">R (GitHub)</a> | <a href="https://CRAN.R-project.org/package=vimp">R (CRAN)</a> | <a href="https://github.com/bdwilliamson/vimpy">Python (GitHub)</a> | <a href="https://pypi.org/project/vimpy/">Python (PyPI)</a> | <a href="https://briandwilliamson.tumblr.com">video and technical slides</a></p>
</div>
<div id="estimating-microbial-abundance" class="section level2">
<h2>Estimating microbial abundance</h2>
<p>In this work, we propose a statistical method for combining inexpensive (but often lower-quality) estimates of relative abundance (e.g., collected using universal primers for the 16S rRNA gene) and higher quality (but often expensive) estimates of absolute abundance (e.g., from quantitative PCR) to estimate absolute abundance of all taxa. Crucially, our proposal accounts for differences in the efficiency of taxon detection in the relative and absolute abundance data. This is joint work with <a href="http://faculty.washington.edu/jphughes/">Jim Hughes</a> and <a href="http://statisticaldiversitylab.com/team/amy-willis">Amy Willis</a>.</p>
<p><a href="https://biorxiv.org/cgi/content/short/761486v1">preprint</a> | <a href="https://github.com/statdivlab/paramedic">GitHub</a></p>
</div>
<div id="clinical-trials" class="section level2">
<h2>Clinical trials</h2>
<p>In settings where it is not ethical to perform a classical placebo-controlled trial, new interventions must be evaluated against the standard of care. Non-inferiority trials are often used to evaluate new interventions in this context. Here, the goal is for the experimental intervention to be “not unacceptably worse” than the standard of care, so that there are an increased number of viable intervention options. A common assumption in non-inferiority trials is that the active control population (typically receiving standard of care) is similar to the populations in prior placebo-controlled randomized trials. However, if characteristics in the current trial differ from previous trials – for example, adherence to the active control regimen is lesser or greater than planned – then the chance of a false positive result (type I error) or a false negative result (type II error) can be different than planned. In this work, we proposed a method for rigorously adapting a non-inferiority trial based on observed characteristics of the active control participants. This is joint work with <a href="https://www.fredhutch.org/en/labs/profiles/donnell-deborah.html">Deborah Donnell</a>, <a href="https://www.fredhutch.org/en/labs/profiles/hanscom-brett.html">Brett Hanscom</a>, and Jim Hughes.</p>
<p><a href="https://journals.sagepub.com/doi/10.1177/0962280218801134#">journal</a> | <a href="https://www.ncbi.nlm.nih.gov/pubmed/30293490">PubMed</a></p>
</div>
</div>
<div id="collaborative-science" class="section level1">
<h1>Collaborative science</h1>
<div id="hiv-vaccines" class="section level2">
<h2>HIV vaccines</h2>
<div id="broadly-neutralizing-antibodies" class="section level3">
<h3>Broadly neutralizing antibodies</h3>
<p>The broadly neutralizing antibody VRC01, developed by the Vaccine Research Center of the National Institute of Allergy and Infectious Diseases, is currently being evaluated in the <a href="https://ampstudy.org/">Antibody Mediated Prevention</a> study (AMP). While the primary goal of AMP is to assess the preventative efficacy of VRC01, a key secondary goal is to determine if this efficacy differs based on characteristics of the HIV virus. There are many possible locations on the HIV genome that we might be interested in; if we simply did a statistical test for association at each possible location, and then appropriately accounted for this multiple testing, then we would have low power to detect effects. To prepare for AMP, we used data on HIV viruses from the <a href="https://www.hiv.lanl.gov/components/sequence/HIV/neutralization/main.comp">Compile, Neutralize, and Tally NAb Panels</a> database. Our goals were to: (i) develop a score that performed well for predicting whether or not an HIV virus was sensitive to neutralization by VRC01, and (ii) to identify groups of HIV genome features that were important in predicting neutralization sensitivity.</p>
<p><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006952">journal</a> | <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Prediction+of+VRC01+neutralization+sensitivity+by+HIV-1+gp160+sequence+features">PubMed</a> | <a href="https://github.com/benkeser/vrc01">GitHub</a> | <a href="https://www.fredhutch.org/en/news/spotlight/2019/05/vidd_magaret_ploscompbio.html">Fred Hutchinson Cancer Research Center Science Spotlight</a></p>
<p>We are currently working on extending the methods used for the VRC01 analysis to combinations of antibodies – a prevention strategy involving multiple bnAbs is likely to be much more effective than a strategy with a single bnAb. Both projects are joint work with <a href="https://www.linkedin.com/in/camagaret/">Craig Magaret</a>, <a href="https://www.benkeserstatistics.com/#!">David Benkeser</a>, Peter Gilbert, and others.</p>
</div>
<div id="correlates-of-risk-and-protection" class="section level3">
<h3>Correlates of risk and protection</h3>
<p>While we do not yet have a broadly efficacious vaccine against HIV-1, clinical trials of HIV-1 vaccine candidates can identify potential correlates of risk or of protection. These correlates are measurable signs that a person is at risk of or is protected from infection by HIV-1, respectively. In this work, I collaborated with Peter Gilbert and others affiliated with the <a href="https://www.hvtn.org/en.html">HIV Vaccine Trials Network</a> to analyze data from the <a href="https://www.hvtn.org/en/community/community-compass/vol19-issue1/hvtn-505-recap.html">HVTN 505</a> clinical trial. We identified three antibody features that inversely correlated with HIV-1 infection risk. This may help in designing future HIV-1 vaccines.</p>
<p><a href="https://doi.org/10.1172/JCI126391">journal</a> | <a href="https://doi.org/10.1172/JCI132535">commentary</a> | <a href="https://www.fredhutch.org/en/news/spotlight/2019/11/neidich_vidd_jci.html">Fred Hutchinson Cancer Research Center Science Spotlight</a></p>
</div>
</div>
<div id="hiv-prevention" class="section level2">
<h2>HIV prevention</h2>
<div id="characteristics-associated-with-hiv-risk" class="section level3">
<h3>Characteristics associated with HIV risk</h3>
<p>This study is based on data from the HIV Prevention Trials Network (HPTN) trial 068. In this work, we aimed to identify factors associated with age-discordant partnerships (one partner greater than 5 years older than the other) and examine the association between partner age discordance and HIV risk in young South African women. We found that a history of age-discordant partnerships is associated witih greater odds of reporting HIV risk factors. This is joint work with <a href="https://fmch.duke.edu/profile/tiarney-ritchwood">Tiarney Ritchwood</a>, Jim Hughes, and others.</p>
<p><a href="https://insights.ovid.com/article/00126334-201608010-00010">journal</a> | <a href="https://www.ncbi.nlm.nih.gov/pubmed/26977748">PubMed</a></p>
</div>
<div id="hiv-treatment-as-prevention" class="section level3">
<h3>HIV Treatment as Prevention</h3>
<p>HPTN 063 was a clinical trial that aimed to examine potential HIV transmissions and the frequency of bacterial sexually transmitted infection (STI) acquisition among an international group of HIV-positive individuals in HIV care. The long-term goal was to inform treatment-as-prevention programs and provide ways of evaluating the potential impact of such programs. We found that there were substantial numbers of both estimated HIV transmissions and STIs, despite sampling individuals in HIV care. This suggested that augmenting secondary prevention interventions may be a means to decrease HIV incidence. This is joint work with <a href="https://people.miami.edu/profile/sas436@miami.edu">Steven Safren</a>, Jim Hughes, and others.</p>
<p><a href="https://onlinelibrary.wiley.com/doi/full/10.7448/IAS.19.1.21096">journal</a> | <a href="https://www.ncbi.nlm.nih.gov/pubmed/27687145">PubMed</a></p>
</div>
</div>
<div id="cancer-treatment" class="section level2">
<h2>Cancer treatment</h2>
<div id="optimizing-cancer-treatment-strategies-using-single-cell-data" class="section level3">
<h3>Optimizing cancer treatment strategies using single-cell data</h3>
<p>In this work, our goal was to develop an algorithm for optimally assigning patients to cancer treatment strategies. We leveraged data from single-cell mass cytometry to: automatically determine subpopulations of cells; create a nested effects model to infer gene hierarchy based on perturbation effects due to administered drugs; and create a scoring and ranking algorithm that identifies the minimal number of drugs among a combination of drugs with teh maximal desired intracellular effects. This is joint work with <a href="http://med.stanford.edu/plevritis.html">Sylvia Plevritis</a>, <a href="http://med.stanford.edu/plevritis.html/People">Benedict Anchang</a>, and others.</p>
<p><a href="https://www.pnas.org/content/115/18/E4294">journal</a> | <a href="https://www.pnas.org/content/pnas/115/18/E4294.full.pdf">pdf</a></p>
</div>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
